app/webpack/stylesheets/components/_panels.scss

```diff
@@ -0,0 +1,30 @@
+.supplier-numbers-wrapper .panel .form-group,
```

Comment:
@jsugarman you can remove this altogether and use the Details component instead: 
https://design-system.service.gov.uk/components/details/

Reply:
thats done now

Reply:
removed the commit with this css and manifest change